app/src/main/java/org/wikipedia/nearby/NearbyFragment.kt

```diff
@@ -172,6 +186,41 @@ class NearbyFragment : Fragment(), LinkPreviewDialog.Callback {
         }
     }
 
+    private fun setUpSymbolManagerWithClustering(mapboxMap: MapboxMap, style: Style) {
+        val clusterColorLayers = arrayOf(
+            androidx.core.util.Pair(
+                0, ContextCompat.getColor(requireActivity(), ResourceUtil.getThemedAttributeId(requireContext(), R.attr.success_color))
+            )
+        )
+        val clusterOptions = ClusterOptions()
+            .withClusterRadius(60)
+            .withColorLevels(clusterColorLayers)
+            .withTextSize(literal(12f))
+            .withTextField(Expression.toString(get(POINT_COUNT)))
+            .withTextColor(Expression.color(fetchAttributeId(R.attr.paper_color)))
```

Comment:
We should use `ResourceUtil.getThemedColor(requireContext(), R.attr.paper_color)` instead and I think it should be fine if we are using the `getThemedColor` instead of creating a new method of referencing it.

Reply:
Have changed this but I anticipate many dynamic colors like this by the end so had created a single place for it. If this is more readable thats fine.